backend/de.metas.serviceprovider/src/main/java/de/metas/serviceprovider/everhour/EverhourImporterService.java

```diff
@@ -114,8 +121,8 @@ public void importTimeBookings(@NonNull final ImportTimeBookingsRequest request)
 					.map(interval -> buildGetTeamTimeRecordsRequest(request.getAuthToken(), interval))
 					.map(everhourClient::getTeamTimeRecords)
 					.flatMap(List::stream)
-					.filter(timeRecord -> isGithubID(timeRecord.getTask().getId()))
-					.forEach(timeBooking-> importTimeBooking(timeBooking, request.getOrgId()));
+					.filter(timeBooking -> TimeBookingTask.TimeBookingTaskId.of(timeBooking.getTask().getId()).isGithubTask())
```

Comment:
@fp-ps  on master, i had to hotfix a NPE at this line: https://github.com/metasfresh/metasfresh/pull/13518.

Pls make sure we won't get it back.

Reply:
I merged `master` into the current branch to have the latest changes